pkg/storage/permanent/permanent.go

```diff
@@ -100,6 +102,14 @@ func (p *Permanent) Accounts(optRealm ...byte) kvstore.KVStore {
 	return lo.PanicOnErr(p.accounts.WithExtendedRealm(optRealm))
 }
 
+func (p *Permanent) LatestNonEmptySlot(optRealm ...byte) kvstore.KVStore {
+	if len(optRealm) == 0 {
+		return p.accounts
```

Comment:
copy paste error I guess?
```suggestion
		return p.latestNonEmptySlot
```

Reply:
Yep!